shard_consumer.go

```diff
@@ -216,24 +211,10 @@ mainloop:
 					// retry infinitely after expired iterator is renewed successfully
 					continue mainloop
 				}
-
-				// Only retry for errors that should be retried; notably, don't retry serialization errors because something bad is happening
-				shouldRetry := request.IsErrorRetryable(err) || request.IsErrorThrottle(err)
```

Comment:
Did we remove some retry logic here?

Reply:
Oh I mentioned that in the original PR https://github.com/snowplow-devops/kinsumer/pull/26#discussion_r2090413786.

Reply:
OK. I'm happy with that